cmd/ocm/gcp/gcp-client-shim.go

```diff
@@ -542,18 +545,19 @@ func (c *shim) undeleteRole(
 // EnsurePolicyBindingsForProject ensures that given roles and member, appropriate binding is added to project.
 // Roles should be in the format projects/{project}/roles/{role_id} for custom roles and roles/{role_id}
 // for predefined roles.
+// Return value indicates whether a modification occurred.
 func (c *shim) ensurePolicyBindingsForProject(
 	ctx context.Context,
 	roles []string,
 	member string,
 	projectName string,
-) error {
+) (bool, error) {
 	needPolicyUpdate := false
 
 	policy, err := c.gcpClient.GetProjectIamPolicy(ctx, projectName, &cloudresourcemanager.GetIamPolicyRequest{})
 
 	if err != nil {
-		return fmt.Errorf("error fetching policy for project: %v", err)
+		return false, fmt.Errorf("error fetching policy for project: %v", err)
```

Comment:
Not related specifically to this MR: why not using errors.Wrap (as done in other places in this file)?